src/test/java/com/puppycrawl/tools/checkstyle/XpathRegressionTest.java

```diff
@@ -0,0 +1,505 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
> base on #5648 (comment)

probably I missed this comment from you
Do you want me to make separate test classes for each check or it is ok to have one big test file?

Reply:
@timurt ,

after conversation with @rnveach : 
please just move this Test to `it` folder, as one test class.
In future .... ones we end up with this file to be huge we can refactor it to multiple test classes grouped by Check package number, example `XpathRegressionCodingTest.java`.

Expected path:
`checkstyle/src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionTest.java`

`org/checkstyle` because own this domain, may be one day in far future we will migrate to new package names.

Reply:
done